bitnami/wordpress/templates/ingress.yaml

```diff
@@ -21,7 +23,7 @@ spec:
   {{- end }}
   rules:
     {{- if .Values.ingress.hostname }}
-    - host: {{ .Values.ingress.hostname | quote }}
+    - host: {{ tpl .Values.ingress.hostname $baseContext | quote }}
```

Comment:
```suggestion
    - host: {{ tpl .Values.ingress.hostname $ | quote }}
```

[When execution begins, $ is set to the data argument passed to Execute, that is, *to the starting value of dot.](https://pkg.go.dev/text/template)

Reply:
Shouldn't this apply to all other $baseContext entries for this file? @andreas-eberle

Reply:
indeed :)

Reply:
True, sorry guys. I had another look at it and in all but one case, I can even use the normal .
I updated the branch. Hopefully its good now :)

Reply:
I think there could be a misunderstanding here: 

Using $ instead of baseContext in all the occurrences in this diff would be the best solution, since `$` saves the context of `.` at the start of the templating, so you don't need actually to use a auxiliar variable `$baseContext`to save the `.` context: it is the same as `$`.

`- host: {{ tpl .Values.ingress.hostname $ | quote }}`

would work here.